src/mappings/Cache.ts

```diff
@@ -154,7 +154,7 @@ export async function getPoolMembers(blockId: number) : Promise<[string, PalletN
     const members: [string, PalletNominationPoolsPoolMember][] = (await api.query.nominationPools.poolMembers.entries()).filter(
         ([_, member]) => member.isSome
     ).map(
-        ([accountId, member]) => [accountId.toString(), member.unwrap()]
+        ([accountId, member]) => [accountId.args.toString(), member.unwrap()]
```

Comment:
I suggest to rename `accountId` to somethening like `storageKeys` to properly reflect the type
Also affair `args` should be a tuple so its worth specifying which specific argument we're using `args[0]`